internal/setup/setup_test.go

```diff
@@ -1150,6 +1151,46 @@ func runParseReleaseTests(c *C, tests []setupTest) {
 	}
 }
 
+// Test YAML marshalling of setup.Package and setup.Slice
```

Comment:
[Nit]: I would drop the comment, the test name should be enough in this case.

Reply:
Dropped in https://github.com/canonical/chisel/pull/101/commits/eb20870d13adf6b28055c7eca9b6259951c056f9.